lib/shelley/bench/Restore.hs

```diff
@@ -309,12 +309,10 @@ cardanoRestoreBench tr c socketFile = do
         , benchRestoreMultipleWallets 100 (unsafeMkPercentage 0.01)
 
         , benchRestoreSeqWithOwnership (Proxy @0)
-        , benchRestoreSeqWithOwnership (Proxy @1)
-        , benchRestoreSeqWithOwnership (Proxy @4)
+        , benchRestoreSeqWithOwnership (Proxy @2)
 
         , benchRestoreRndWithOwnership (Proxy @0)
-        , benchRestoreRndWithOwnership (Proxy @1)
-        , benchRestoreRndWithOwnership (Proxy @4)
+        , benchRestoreRndWithOwnership (Proxy @2)
```

Comment:
It's a little bit confusing to have the address discovery ratio as a unitless type parameter, but now I see this means 0% and 0.2%.

Probably just one or two percentage benchmarks is enough anyway.

Next thing we should do is improve the performance so we can increase these numbers. :smile:.

And also to benchmark the sequential address pool code in isolation.